### PR TITLE
feat: add missing requests

### PR DESCRIPTION
**Pull request by peterdeme**
- Full feature parity for Java

### CODE REVIEW

File: src/main/java/io/getstream/chat/java/services/UserService.java [hunk 1]
```diff
@@ -64,5 +64,6 @@
   Call<StreamResponseObject> unban(
       @NotNull @Query("target_user_id") String targetUserId,
       @Nullable @Query("type") String channelType,
-      @Nullable @Query("id") String channelId);
+      @Nullable @Query("id") String channelId,
+      @Nullable @Query("shadow") Boolean shadow);
```
Comment (driver-devel) [inline src/main/java/io/getstream/chat/java/services/UserService.java:68]: Seems like this param doesn't have any effect. It removes 2 types of bans. @ferhatelmas

Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/services/UserService.java:68]: interesting. I couldn't find it in the REST docs, but Python/Ruby etc has it

Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/services/UserService.java:68]: https://github.com/GetStream/stream-chat-ruby/blob/602802956c475c2fdfcd168aafff70edfb042d19/lib/stream-chat/client.rb#L164

Comment (driver-devel) [reply src/main/java/io/getstream/chat/java/services/UserService.java:68]: I see go sdk has it too 🤔. Alright, let's keep it for consistency

**Review by driver-devel — APPROVED** [review]

File: src/main/java/io/getstream/chat/java/services/UserService.java (hunk 1 above)
Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/services/UserService.java:68]: and the tests succeed as well, so.. 🤔

**Pull request merged**